mappings/net/minecraft/world/TimeAccess.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5424 net/minecraft/world/TimeAccess
```

Comment:
Not a big fan of this, I would prefer that all world classes be explicitly clear that they are a world. How about `DimensionWorldView` or something similar, because it deals with dimension related properties?

Reply:
I looked into that interface for a bit and it seems to be pretty moon/lunar cycle heavy. I'll probably open a PR in a bit documenting the whole thing